src/cmds/scripts/pbs_habitat.in

```diff
@@ -59,6 +59,12 @@ get_db_user() {
 }
 
 chk_dataservice_user() {
+
+	# verify that postgres has been initalized correctly
+	if [ ! -d /var/lib/pgsql/ ]; then
```

Comment:
The current code checks and creates a database inside the pg_db_utility functions inside libdb. Please check this: https://github.com/openpbs/openpbs/blob/b1a41e3bfd2137d5eef006fb59808e2d311434f1/src/lib/Libdb/pgsql/pbs_db_utility#L378

That might be a more appropriate location to update/enhance.